SU-SWS/stanford_bean_types
# 7.x 2.x dev jgk

# READY FOR REVIEW

# Summary
- Add new image insert option to stanford_large_block

# Needed By (Date)
- Today

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that you can insert the Large Landscape Scaled image style on stanford_large_block

# Affected Projects or Products
- SOE
- stanford_bean_types

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-2030

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
